documentation/README.md

```diff
@@ -21,12 +26,9 @@ The SDK is published as a NuGet package under the name [Microsoft.Performance.SD
 Since it is hosted on [NuGet.org](https://www.nuget.org/), it can be added to a `csproj` with no additional configuration by using 
 the Visual Studio NuGet Package Manager, `dotnet.exe`, or `nuget.exe`.
 
-# Creating A Plugin
-Refer to [Creating your first plugin](Using-the-SDK/Creating-your-plugin.md)
-
 # Recommended Reading Order
 To best understand how the SDK works and how to develop SDK plugins, it is recommended to read documentation in the following order:
-1) [Architecture/Overview](./Architecture/Overview.md) to understand at a high level the various systems the SDK provides
-2) [Architecture/The Data-Processing Pipeline](./Architecture/The-Data-Processing-Pipeline.md) to understand how to systematically process data that 
+1. [Architecture/Overview](./Architecture/README.md) to understand at a high level the various systems the SDK provides
```

Comment:
Wondering if we should make the display text say "Architecture/README" if we want it to be a path or just "Overview" for a better description.

Reply:
The overview describes the file contents, README.md is the file name because it will show up on github if you navigate to that folder. I don't think they need to match.